apps/actors/lib/actors/actor/entity.ex

```diff
@@ -576,6 +514,64 @@ defmodule Actors.Actor.Entity do
     %{state | actor: %{actor | state: new_state}}
   end
 
+  defp invoke_host(
+         %ActorInvocation{actor_name: name, actor_system: system, command_name: command} =
+           payload,
+         %EntityState{
+           actor: %Actor{state: actor_state}
+         } = state
+       ) do
+    if Enum.member?(@default_methods, command) do
+      {context, current_state} =
+        case actor_state do
+          nil ->
+            {Context.new(), nil}
+
+          _ ->
+            %ActorState{state: current_state} = actor_state
+            {Context.new(state: current_state), current_state}
+        end
+
+      resp =
+        ActorInvocationResponse.new(
+          actor_name: name,
+          actor_system: system,
+          updated_context: context,
+          value: current_state
+        )
+
+      {:reply, {:ok, resp}, state}
+    else
+      payload
+      |> ActorInvocation.encode()
+      |> Actors.Node.Client.invoke_host_actor()
```

Comment:
I don't get it, why default methods dont invoke host actor too?

Reply:
Because standard methods are just standard and that's exactly why they don't need to be created by default in the SDKs. 
As they are standards and therefore aware of the expected behavior they can do all the work still in the Proxy layer.

Marcel and I have added new information to the main thread so you can capture motivations there if you wish. Link here
https://github.com/eigr/spawn/discussions/56